### PR TITLE
formik integration added

### PR DESCRIPTION
**Pull request by mubeenfayyaz20**

### CODE REVIEW

File: src/pages/EventManagement/AddNewEvents.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import EventForm from "../../components/EventForm";
 
 const AddNewEvents = () => {
   const apiCall = async (data) => {
+    debugger;
```
Comment (ahsanbilalas) [inline src/pages/EventManagement/AddNewEvents.jsx]: remove debugger;

File: src/pages/EventManagement/AllEvents.jsx [hunk 2]
```diff
@@ -24,9 +24,11 @@ const AllEvents = () => {
   };
 
   const deleteEvent = (id) => {
-    const deleteData = eventDelete(id);
+    debugger;
```
Comment (ahsanbilalas) [inline src/pages/EventManagement/AllEvents.jsx]: remove debugger;

**Review by ahsanbilalas — APPROVED** [review]

File: src/pages/EventManagement/AddNewEvents.jsx (hunk 1 above)
Comment (mubeenfayyaz20) [reply src/pages/EventManagement/AddNewEvents.jsx]: done

File: src/pages/EventManagement/AllEvents.jsx (hunk 2 above)
Comment (mubeenfayyaz20) [reply src/pages/EventManagement/AllEvents.jsx]: done

**Pull request merged**